### PR TITLE
Add a larger dictionary to Liedle closes #19

### LINKED ISSUE #19: Add dictionary
Add a list of at least 1000 five-letter English words to pick secret words from. Add a larger list of five-letter English words that the user can use as valid guesses.

### PR DESCRIPTION
**Pull request by Destiny-02**
- Dictionary is stored and exported from a separate file `dictionary.js`
- Dictionary contains ~6000 5-letter English words that are considered "valid" guesses in Liedle
- Dictionary source was added to README (https://www-cs-faculty.stanford.edu/~knuth/sgb.html)

### CODE REVIEW

File: src/liedle/dictionary.js [hunk 1]
```diff
@@ -0,0 +1,5758 @@
...
+    "batty",
+    "coifs",
+    "suety",
+    "gabby",
+    "hafta",
+    "pitas",
+    "gouda",
+    "deice",
+    "taupe",
+    "topes",
```
Comment (k1dubv1) [inline src/liedle/dictionary.js:4563]: Starting from about line 2000 onwards the words start to get less and less commonly known and from about line 3000 onwards an average person probably won't know that most of these words exist. The word list might not be very suitable for Liedle.

**Review by k1dubv1 — COMMENTED** [review]
The code seems fine but just a bit unsure on the dictionary itself.

**Review by k1dubv1 — CHANGES_REQUESTED** [review]
Just the comments above. However, if you're fine with how it is then I'll approve it.

File: src/liedle/dictionary.js (hunk 1 above)
Comment (Destiny-02) [reply src/liedle/dictionary.js:4563]: One of the things I like about playing Wordle/Liedle is learning new words. Sure, it's frustrating when the answer is something bizarre, but I think it adds to the fun/luck component of the game. Maybe in the future, there could be two dictionaries - one to act as a pool of secret words and one to refer to when determining when a guess is a valid word.

**Review by k1dubv1 — APPROVED** [review]
Alright sounds good, makes it interesting and unpredictable

**Pull request merged**